sdk/trace/trace_test.go

```diff
@@ -1261,6 +1261,11 @@ func TestWithResource(t *testing.T) {
 		{
 			name:    "explicitly empty resource",
 			options: []TracerProviderOption{WithResource(resource.Empty())},
+			want:    resource.Environment(),
+		},
+		{
+			name:    "explicitly environment resource",
+			options: []TracerProviderOption{WithResource(resource.Environment())},
 			want:    resource.Empty(),
```

Comment:
This should be `resource.Environment()`, shouldn't it?  Which demonstrates the concern noted about the test case above.  It's not actually proving the code functions correctly.

Reply:
Yes,adding attributes to the OTEL_RESOURCE_ATTRIBUTES environment variable will change the test case